assets/reader-activation/index.js

```diff
@@ -127,12 +116,27 @@ export function off( event, callback ) {
  *
  * @param {string} email Email.
  */
-export function setReader( email ) {
-	store.reader = null;
+export function setReaderEmail( email ) {
```

Comment:
Should be updated [in Reader Registration block](https://github.com/Automattic/newspack-plugin/blob/96a07ae3873d23775da826606582bd1a84342515/assets/blocks/reader-registration/view.js#L30).